.github/advanced-issue-labeler.yml

```
@@ -4,17 +4,15 @@ policy:
   - section:
       - id: ["versions"]
         label:
-          - name: "ver: 1.6.x"
-            keys: ["v1_6", "v1.6", "1.6.x"]
-          - name: "ver: 1.7.x"
-            keys: ["v1_7", "v1.7", "1.7.x"]
-          - name: "ver: 1.8.x"
-            keys: ["v1_8", "v1.8", "1.8.x"]
-          - name: "ver: 1.9.x"
-            keys: ["v1_9", "v1.9", "1.9.x"]
           - name: "ver: 2.0.x"
             keys: ["v2_0", "v2.0", "2.0.x"]
```

```suggestion
          - name: "ver: 1.x"
            keys: ["v1_x", "v1.X", "1.x"]
          - name: "ver: 2.0.x"
            keys: ["v2_0", "v2.0", "2.0.x"]
```
parity with https://github.com/Lightning-AI/pytorch-lightning/pull/20154/files#r1702713254